src/main/java/at/hannibal2/skyhanni/features/combat/FlareDisplay.kt

```diff
@@ -194,7 +194,6 @@ object FlareDisplay {
         SOS("§5SOS Flare", "+125%"),
         ALERT("§9Alert Flare", "+50%"),
         WARNING("§aWarning Flare", null),
-        ;
```

Comment:
this doesnt serve much purpose now but i see no reason to remove it as its probably there due to ktlint before which is planned to be brough back in some sense in the future

Reply:
we want trailing commas everywhere, please leave it